backend/tsconfig.json

```diff
@@ -4,14 +4,30 @@
     "module": "ESNext",
     "moduleResolution": "node",
     "outDir": "./dist",
-    "rootDir": "./src",
```

Comment:
this needs to be ./src so that it's not compiling the entire backend module, just the typescript source files

Reply:
dist should then look like this locally
![image](https://github.com/user-attachments/assets/8d2a73cf-5aa2-4d80-af81-7f915610656f)